velox/exec/Operator.h

```diff
@@ -494,10 +534,14 @@ class OperatorRuntimeStatWriter : public BaseRuntimeStatWriter {
   void addRuntimeStat(const std::string& name, const RuntimeCounter& value)
       override {
     if (operator_) {
-      operator_->stats().addRuntimeStat(name, value);
+      operator_->addRuntimeStat(name, value);
     }
   }
 
+  void setActiveOperator(Operator* FOLLY_NULLABLE op) {
```

Comment:
This can be simplified and we don't need to reuse OperatorRuntimeStatWriter and it can be a on-stack var.

Reply:
Need some exaplanation on this one.